docs/salt.md

```diff
@@ -54,7 +54,7 @@ Key | Value
 `cbr-cidr` | (Optional) The minion IP address range used for the docker container bridge.
 `cloud` | (Optional) Which IaaS platform is used to host kubernetes, *gce*, *azure*, *aws*, *vagrant*
 `etcd_servers` | (Optional) Comma-delimited list of IP addresses the kube-apiserver and kubelet use to reach etcd.  Uses the IP of the first machine in the kubernetes_master role.
-`hostnamef` | (Optional) The full host name of the machine, i.e. hostname -f
+`hostnamef` | (Optional) The full host name of the machine, i.e. uname -n
```

Comment:
It might be more descriptive to say "The network node hostname of the machine" if it is no longer going to be an FQDN.